src/participant/participant.ts

```diff
@@ -645,8 +649,17 @@ export default class ParticipantController {
 
       const schema = await getSimplifiedSchema(sampleDocuments);
       this._schema = new SchemaFormatter().format(schema);
+
+      const useSampleDocsInCopilot = !!vscode.workspace
+        .getConfiguration('mdb')
+        .get('useSampleDocsInCopilot');
+
+      if (useSampleDocsInCopilot) {
+        this._sampleDocuments = formatSampleDocuments(sampleDocuments);
```

Comment:
The PR description says we're passing in 3 sample docs to copilot, but here it appears we're formatting the entire sampleDocuments dataset, which is 4 docs, as far as I can tell.

Reply:
Yeah, the number should be 4. Good catch. Copy and paste from compass.